mobile/src/navigation/HomeStackNavigator.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { StyleSheet, Image } from 'react-native';
+import { createStackNavigator } from '@react-navigation/stack';
+import { RouteProp } from '@react-navigation/native';
+import { HomeScreen } from '../screens/HomeScreen';
+import { BookScreen } from '../screens/BookScreen';
+import { Language } from '../models/Languages';
+
+type HomeStackParams = {
+  Home: undefined;
+  Book: { id: string, langs: Language[] };
```

Comment:
Let's pass the entire Book object into the screen, so we can preload the title, author, and image. You'll still have to call GET /books/id/lang to get the read/explore/learn tabs but at least we can render some content earlier.